### PR TITLE
added staticdocs

### PR DESCRIPTION
**Pull request by kirillseva**

### CODE REVIEW

**Comment by kirillseva** [comment]
Should probably integrate with rocco - add links to staticdocs (stored in inst/web)

File: README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
-Stagerunner [![Build Status](https://travis-ci.org/robertzk/stagerunner.svg?branch=master)](https://travis-ci.org/robertzk/stagerunner) [![Coverage Status](https://coveralls.io/repos/robertzk/stagerunner/badge.png?branch=master)](https://coveralls.io/r/robertzk/stagerunner) [![Documentation](https://img.shields.io/badge/rocco--docs-%E2%9C%93-blue.svg)](http://robertzk.github.io/stagerunner/)
+Stagerunner [![Build Status](https://travis-ci.org/robertzk/stagerunner.svg?branch=master)](https://travis-ci.org/robertzk/stagerunner) [![Coverage Status](https://img.shields.io/coveralls/robertzk/stagrunner.svg)](https://coveralls.io/r/robertzk/stagerunner) [![Documentation](https://img.shields.io/badge/rocco--docs-%E2%9C%93-blue.svg)](http://robertzk.github.io/stagerunner/)
```
Comment (robertzk) [inline README.md]: typo

**Comment by robertzk** [comment]
Yeah, not sure I want this in master. :/

**Comment by kirillseva** [comment]
it's simpler to have this in master. this way `rocco::rocco` should be idempotent, as it can rely on staticdocs being present in a fixed place

**Comment by robertzk** [comment]
Ok, that's fair.

**Pull request merged**